doc/api_repos.rst

```diff
@@ -108,10 +108,25 @@ Repository Configuration
 
     If enabled, DNF will continue running and disable the repository that couldn't be contacted for any reason when downloading metadata. This option doesn't affect skipping of unavailable packages after dependency resolution. The default is ``True``.
 
+  .. attribute:: sslcacert
+
+    Path to the directory or file containing the certificate authorities to verify SSL certificates.
+    Defaults to none - uses system default.
+
   .. attribute:: sslverify
 
     Whether SSL certificate checking should be performed at all. Defaults to ``True``.
 
+  .. attribute:: sslclientcert
+
+    Path to the SSL client certificate used to connect to remote sites.
+    Defaults to none.
```

Comment:
in api_conf and api_repos honor the Pythonic semantic. E.g. write "Defaults to ``None``."